src/views/confirmation.njk

```diff
@@ -31,6 +31,19 @@
         <a href="javascript:if(window.print)window.print()" class="govuk-link">Print a copy of this appeal</a>
       </p>
       <div class="govuk-!-display-none-screen">
+        <h2 class="govuk-heading-l">Penalty Details</h2>
```

Comment:
Is it part of the ticket to add this to the confirmation page?

Reply:
This is for the printout, which is part of the ticket

Reply:
ok, just thought i'd check